### ISSUE TITLE
@MustBeDocumented not working in javadoc format

### ISSUE THREAD

**Issue by athkalia** [Kotlin/dokka#334]
Hello,

Trying to get an understanding of what @MustBeDocumented is doing. It doesn't seem to include my annotation (called @Beta) in the dokka documentation when i use my annotation in a method declaration for example.

the `outputFormat` is `javadoc` for dokka, not sure if that plays a role.

Cheers,
Sakis

**Comment by semoro** [comment]
Please, check output format `html`, it can be possible bug of Javadoc format

**Comment by semoro** [comment]
In `html` format annotation should be included on full method page 
That works for me

**Comment by athkalia** [comment]
I don't quite understand what you are saying, care to explain please?
This is how I am using my annotation:

```
interface MyInterface {

    @Beta
    fun method(): String
}
```

**Comment by semoro** [comment]
There shouldn't be the annotation in method description `-my-interface/index.html` (class summary page), but should be in `-my-interface/method.html`

**Comment by athkalia** [comment]
After doing a clean build It started appearing for `html`, which is good. It's still not appearing on `javadoc` however. Any plans to fix that?

**Comment by semoro** [comment]
No plans yet, since you're first one who's reported it

**Comment by Kordyjan** [comment]
This is fixed in 1.4.0-rc
